views/js/qtiCreator/editor/gridEditor/droppable.js

```diff
@@ -305,8 +305,10 @@ define([
         ).on('mouseleave.gridEdit.gridDragDrop',
             _.debounce(function(e){
                 var goingTo = e.relatedTarget|| e.toElement; //browser compatibility
+                var $target = $(e.target);
 
-                if($(e.target).parents('.qti-itemBody').length) {
+                if($target.parents('.qti-itemBody').length ||
```

Comment:
I'd put also $target && .. just to make sure but if we did not used to, then it's not necessary.

Reply:
No we dont do and over here

Reply:
$target is always here, otherwise there would be no event